okta/api_response.py

```diff
@@ -182,5 +182,6 @@ async def get_next(self):
                 next_response = OktaAPIResponse(
                     self._request_executor, req, res_details, resp_body)
                 self._next = next_response._next
+                self._resp_headers = res_details.headers
```

Comment:
Is `self._resp_headers` used anywhere?

Reply:
That's a good question.  @haggrip are you able to provide further insight here?